backend/resources/contact.py

```diff
@@ -109,11 +112,15 @@ def send_message(self, *, context: dict, subject: str, body: str, **kwargs):
 
         stderr = err()
 
-        if "MSGID=" not in stderr:
+        # example successful responses from ethereal and ses, respectively:
+        # data: (250, b'Accepted [STATUS=new MSGID=YRbuVPOb3gQndRMOYRubW3BtxAZI8iBfAAAAAbdiFY1fwZ4LCwdvaNWoIDs]')
+        # data: (250, b'Ok 0102017b4c5776db-6639a4be-4c46-469b-b528-edf2b28e686a-000000')
```

Comment:
I'm not really sure what it is for?

Reply:
just a reference material, it's easier to take a look at the raw data from commends than set up whole environment (especially the production one) just to check how these responses actually look like inside.